src/main/flight/position.c

```diff
@@ -103,6 +106,7 @@ void calculateEstimatedAltitude(timeUs_t currentTimeUs)
 
     int32_t baroAlt = 0;
     int32_t gpsAlt = 0;
+    uint16_t gpsNumSat = 0;
```

Comment:
```suggestion
    uint8_t gpsNumSat = 0;
```
The number of sats will never exceed 255, (I hope ;)) so we can go smaller.

Reply:
Untill we start using starlink for navigation. Fixed.